pkg/exec/exec.go

```diff
@@ -123,7 +122,7 @@ func execCmdInPod(ctx context.Context, clientsets *k8sutil.Clientsets,
 
 	if containerName == "rook-ceph-tools" {
 		cmd = append(cmd, "--connect-timeout=10")
-	} else if cmd[0] == "ceph" {
+	} else if cmd[0] == "ceph" && args[0] != "daemon" {
```

Comment:
dont we need to check for `args` length before accessing it?

Reply:
we don't want to run the command `ceph daemon *` with the extra flags like `--config` which is required for other commands. That's why I have added this check so that command that starts with `ceph daemon * ` should not have the extra flags

Reply:
my suggestion was to do len(agrs)>1 and return error if its less than 1 to ensure that no panic happens as we are accessing it.

Reply:
I don't why I misinterpreted. We check the length in some places but better to add the check so I added now